blazor/dropdown-list/accessibility.md

```diff
@@ -31,15 +31,20 @@ You can use the following key shortcuts to access the DropDownList without inter
 
 | **Keyboard shortcuts** | **Actions** |
 | --- | --- |
-| <kbd>Arrow Down</kbd> | Selects the first item in the DropDownList when no item is selected. Otherwise, selects the item next to the currently selected item. |
-| <kbd>Arrow Up</kbd> | Selects the item previous to the currently selected one. |
-| <kbd>Page Down</kbd> | Scrolls down to the next page and selects the first item when popup list opens. |
-| <kbd>Page Up</kbd> | Scrolls up to the previous page and selects the first item when popup list opens. |
-| <kbd>Enter</kbd> | Selects the focused item, and when it is in open state, the popup list closes. Otherwise, toggles the popup list. |
+|**Focus**|
+|<kbd>Alt + J</kbd> | Focuses on the first component of the sample. |
+|**Input Navigation**|
+| <kbd>Alt + Down arrow</kbd> | Opens the popup list. |
+| <kbd>Alt + Up arrow</kbd> | Closes the popup list. |
 | <kbd>Tab</kbd> | Focuses on the next TabIndex element on the page when the popup is closed. Otherwise, closes the popup list and remains the focus of the component. |
 | <kbd>Shift + tab </kbd> | Focuses on the previous TabIndex element on the page when the popup is closed. Otherwise, closes the popup list and remains the focus of the component. |
-| <kbd>Alt + Down</kbd> | Opens the popup list. |
-| <kbd>Alt + Up</kbd> | Closes the popup list. |
+|**Selection**|
+| <kbd>Enter</kbd> | Selects the focused item, and when it is in open state, the popup list closes. Otherwise, toggles the popup list. |
+|**Popup Navigation**|
 | <kbd>Esc(Escape)</kbd> | Closes the popup list when it is in an open state and the currently selected item remains the same. |
+| <kbd>Down arrow</kbd> | Selects the first item in the DropDownList when no item is selected. Otherwise, selects the item next to the currently selected item. |
+| <kbd>Up arrow</kbd> | Selects the item previous to the currently selected one. |
+| <kbd>Page down</kbd> | Scrolls down to the next page and selects the first item when popup list opens. |
```

Comment:
| <kbd>Page down</kbd> | Scrolls down to the next page and selects the first item when the popup list opens. |